scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
```

Comment:
Now also checks that mcptam isn't already in src too and kills it and clones a clean copy if it is. Pushed, awaiting CI success.